src/emulator/extensions/postinstall.ts

```diff
@@ -0,0 +1,38 @@
+import { EmulatorRegistry } from "../registry";
+import { Emulators } from "../types";
+
+export function replaceConsoleLinks(postinstall: string): string {
+  const uiInfo = EmulatorRegistry.getInfo(Emulators.UI);
+  const uiUrl = uiInfo ? `http://${EmulatorRegistry.getInfoHostString(uiInfo)}` : "unknown";
+  let subbedPostinstall = postinstall;
+  const linkReplacements = new Map<RegExp, string>([
+    [
+      /(http[s]?:\/\/)?console\.firebase\.google\.com\/project\/[A-Za-z0-9-]+\/storage[A-Za-z0-9\/-]*(?=[\)\]\s])/,
```

Comment:
lint nit: Unnecessary escape character: \/

Reply:
/ does need to be escaped. Otherwise it indicates the end of the regex.